components/raftstore/src/store/worker/split_config.rs

```diff
@@ -87,6 +107,15 @@ impl SplitConfig {
                 ("sample_num should be less than qps_threshold for load-base-split.").into(),
             );
         }
+        if self.grpc_thread_cpu_overload_threshold_ratio > 1.0
```

Comment:
How much should these thresholds be adjusted if we want to turn this feature off?